openupgradelib/openupgrade.py

```diff
@@ -2558,3 +2551,22 @@ def convert_to_company_dependent(
                 )
             ), args,
         )
+
+
+def _load_apriori(path=None):
+    """ Load a 14.0+ Apriori object from apriori.json in the root of the
+    --upgrade-path """
+    if path is None:
+        from odoo import tools
+        path = tools.config['upgrade_path']
+    apriori_path = os.path.join(path, "apriori.json")
+    try:
+        with open(apriori_path) as json_file:
+            apriori_dict = json.load(json_file)
```

Comment:
```suggestion
            apriori_dict = yaml.safe_load(json_file)
```

:laughing: Well you know already probably, but just cross-linking https://github.com/legalsylvain/server-tools/pull/4#pullrequestreview-545971834 and https://github.com/legalsylvain/openupgrade-framework/pull/4#discussion_r537365871.

Reply:
Yes, the JSON was ugly. We're reverting to 'openupgrade_framework as an odoo module' and we can now import apriory as a Python module again.